.github/workflows/e2e.yml

```diff
@@ -75,8 +75,11 @@ jobs:
             packages.microsoft.com:443
             download.cypress.io:443
             cdn.cypress.io:443
-            foresight.service.thundra.io
-            foresight-cli-test-artifact-prod.s3-accelerate.amazonaws.com
+            foresight.service.thundra.io:443
+            foresight-cli-test-artifact-prod.s3-accelerate.amazonaws.com:443
```

Comment:
Thanks for pointing this out. I will keep this in mind in future to specify the port number :)